scripts/create-aws-resources.py

```diff
@@ -0,0 +1,153 @@
+# -*- coding: utf-8 -*-
+from __future__ import print_function
+from __future__ import unicode_literals
+from __future__ import division
+
+import json
+from os import environ
+
+from django.conf import settings
+import boto3
+import botocore
+
+AWS_ACCESS_KEY_ID = environ.get('AWS_ACCESS_KEY_ID')
+AWS_SECRET_ACCESS_KEY = environ.get('AWS_SECRET_ACCESS_KEY')
+# TODO REGION needs to get into an environment variable
+REGION = 'us-east-1'
+
+SQS_QUEUE = 'raster-foundry-lewfish-monoshone'
+SQS_DEAD_LETTER_QUEUE = 'raster-foundry-lewfish-monoshone-dead-letter'
+S3_BUCKET = 'raster-foundry-lewfish-monoshone'
+
+# SQS_QUEUE = environ.get('RF_SQS_QUEUE')
+# SQS_DEAD_LETTER_QUEUE = environ.get('RF_SQS_DEAD_LETTER_QUEUE')
+# S3_BUCKET = environ.get('RF_SQS_DEAD_LETTER_QUEUE')
+
+VISIBILITY_TIMEOUT = 120
+MAX_RECEIVE_COUNT = 5
+
+def get_sqs_policy(q_arn, bucket_name):
+    """
+    Policy that allows S3 bucket to publish to SQS queue.
+    from http://docs.aws.amazon.com/AmazonS3/latest/dev/
+    ... ways-to-add-notification-config-to-bucket.html
+    """
+
+    return {
+        "Version":"2008-10-17",
+        "Id":"example-ID",
+        "Statement": [
+            {
+                "Sid":"example-statement-ID",
+                "Effect":"Allow",
+                "Principal": {
+                    "AWS":"*"
+                },
+                "Action": [
+                    "SQS:SendMessage"
+                ],
+                "Resource": q_arn,
+                "Condition": {
+                    "ArnLike": {
+                        "aws:SourceArn":"arn:aws:s3:*:*:" + bucket_name
+                    }
+                }
+            }
+        ]
+    }
+
+
+class CreateAwsResources(object):
+    """"
+    Encapsulates a process for creating an SQS queue,
+    an attached dead letter queue, and an S3 bucket
+    that sends messages to the SQS queue on object creation.
+    """
+    def __init__(self):
+        self.session = boto3.session.Session(
+            aws_access_key_id=AWS_ACCESS_KEY_ID,
```

Comment:
You should check that leaving this out still works. In other instances where we use boto, it is able to pick up this information directly from the user's aws credentials in the profile.